src/components/toolbar/GradientGenerator.svelte

```
@@ -1,155 +1,215 @@
-
-    function removeStop(index: number) {
-        onRemoveGradientStop(index);
-    }
-
-    function toggleGradientType(type: string) {
-        if (type !== "linear" && type !== "radial") {
-            throw new Error("Invalid gradient type");
-        }
-        onSetGradientType(type as IconGradient["type"]);
-    }
+	import _ld from "lodash";
```

ESLint

[error] '_ld' is defined but never used.

(@typescript-eslint/no-unused-vars)